plugins/idea/build.gradle

```
@@ -26,6 +26,7 @@ dependencies {
 
 // See https://github.com/JetBrains/gradle-intellij-plugin/
 intellij {
+    version = '2021.3.3'
```

```suggestion
    version = project.findProperty('intellijVersion') ?: '2021.3.3'
```
while at it, you could also update the docs on `intellijVersion` [here](https://github.com/openjdk/jtreg/blob/master/plugins/idea/README.md?plain=1#L41)